src/main/java/com/dataloom/datastore/edm/controllers/EdmController.java

```diff
@@ -143,15 +146,28 @@ private void updateEntitySetIdsToGet(
             Set<UUID> propertyTypeIds,
             Set<UUID> entityTypeIds,
             Set<UUID> entitySetIds ) {
+        boolean setRetrieved = false;
```

Comment:
I haven't had the chance to think about it too deeply, but we should mark this for attention post-Jan 24th.